autogen

```diff
@@ -314,6 +314,15 @@ function autogenForFile() {
   local file="$1"
   case "${file}" in
 
+    *.bat)
+      printLicenseNonHashComment "rem "
+      echo
+      if [ ${TOP_LEVEL_COMMENT} -eq 1 ]; then
```

Comment:
Could you elaborate a little bit on why we need TODO for top level comment?

Reply:
This is the default for all languages, including this one. You can skip it with the `--no-tlc` (tlc = top level comment) flag.

Why it's there: `autogen` launched with the top-level-comment feature by default. A few folks asked to make it optional, so I added the `--no-tlc` flag. To remove it, and make it off by default would be a behavior change — I'm not sure how many folks out there are relying on this fact, or whether they all prefer to avoid it with the `--no-tlc` flag, so this is to just keep compatibility.

Do you think it's not worth having a TODO to describe the purpose of the file?

Reply:
FWIW, I'll merge this PR so that you have the ability to easily create *.bat files with the correct `rem` comments. Let's discuss separately whether top-level comments should be on or off by default, as that will affect all languages, not just this one, and it's best to have consistency.